### DISCUSSION TITLE
Neunte Woche

### DISCUSSION THREAD

**Discussion by davidlzt** [MelonigeMelone/xCalendar#23] [Blogkanal]
Diese Woche haben wir die Dokumentation von arc42 erstellt. Dabei haben wir uns an der Struktur von arc42 entlangehangelt.
Wir erstellten unter anderem Qualitätsziele, Komponentendiagramme und Paketdiagramme, Sequenzdiagramme, Architekturentscheidungen und einen Qualitätsbaum und die Qualitätsszenarien. Dazu haben wir immer kleine Beschreibungen hinzugefügt.
Das Dokument findet ihr unter folgendem Link:
[arc42-xCalendar.pdf](https://github.com/MelonigeMelone/xCalendar/files/13587260/arc42-xCalendar.pdf)

**Comment by Christian-2003** [comment]
Hallo zusammen,

ich habe mir euer Dokument angesehen und es scheint ja schon ziemlich gut gefüllt zu sein. Inhaltlich hätte ich keine Fragen diesbezüglich.
Da ihr hier das PDF-Template verwendet habt, wäre es vielleicht schön, wenn alle Abbildungen einen weißen Hintergrund hätten, das ist bei der Bausteinsicht nämlich nicht der Fall, sodass diese etwas stark heraussticht und visuell nicht gut ins Dokument passt.

Ansonsten habt ihr natürlich für die nächste Woche noch einiges im Template auszufüllen, ich bin gespannt wie das Dokument nach der nächsten Woche aussehen wird.
Bezüglich der Qualitätsziele wäre es vielleicht ganz schön, wenn diese neben dem Qualitätsbaum noch genauer erläutert werden würden.

Hoffentlich habt ihr diese Woche auch einiges implementieren können, da würde mich euer aktueller Stand interessieren.

Ich freue mich schon auf die nächste Woche.

**Comment by cool2722** [comment]
Hallo zusammen,

Eure Projekt Dokumentation war sehr detaliert aber wir haben kleine Anmerkungen dazu, und zwar-

1. Ihr habt nur 2 Qualitätsziele aufgeschrieben obwohl den Template 3 bis 5 empfohlen hat.
2. In den Bausteinsicht, ihr habt Klassendiagramm eingefügt statt Komponenten- bzw. Paketdiagramme.
3. Habt ihr ein Design Pattern bezüglich der Architekturentscheidung?


Ansonsten habt ihr sehr gut gemacht und wir freuen uns auf eure Ergebnis!

**Comment by JanWilfert** [comment]
Ich kann den beiden nur zustimmen und habe nichts weiter hinzuzufügen. Insgesamt sieht das alles sehr gut aus, weiter so 👍

**Comment by Charlotte0999** [comment]
Wir finden, dass das Dokument gut gefüllt wurde und man sieht, wie viel Mühe und Gedanken ihr euch beim erstellen gemacht habt.
